tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -61,29 +59,43 @@ class Game extends Component {
       );
     });
     const status = winner ? `Winner: ${winner}` : `Next player: ${this.props.xIsNext ? 'X' : 'O'}`;
-
+    const { constLineLoading, constLineError } = this.props;
     return (
-      <div className="game">
-        <div className="game-board">
-          <Board squares={current.squares} onClick={this.handleClick} />
-        </div>
-        <div className="game-info">
-          <div>{status}</div>
-          <ol>{moves}</ol>
-        </div>
-      </div>
+      <Fragment>
+        {constLineError ? (
```

Comment:
constLineError and constLineLoading can't be true at the same time. So you can write

{ constLineError && <span>{constLineError}</span> }
<div className="game">
  {  constLineLoading ? (
       <span>Cargando...</span>
       ) : (
        <div className="game-board">
           ....
        </div>
  }
</div>